CHANGELOG.md

```diff
@@ -2,6 +2,53 @@
 
 All notable changes to this project will be documented in this file.
 
+> _**notes**_
+> OpenVINO Training Extension which version is equal or older then v0.5.0 does not include the latest functional and security updates. OTE Version 1.0.0 is targeted to be released in February 2023 and will include additional functional and security updates. Customers should update to the latest version as it becomes available.
+
+## \[v1.0.0\]
+
+### New features
+
+- Adaptation of [Datumaro](https://github.com/openvinotoolkit/datumaro) component as a dataset interface
+- Integrate hyper-parameter optimizations
+- Support action recognition task
+- Self-supervised learning mode for representational training
+- Semi-supervised learning mode for better model quality
+
+### Enhancements
+
+- Installation through PyPI - Named as [OTX](https://pypi.org/project/otx/)
+- CLI update - Update `find` command to find configurations of tasks/algorithms - Introduce `build` command to customize task or model configurations - Automatic algorihm selection for the `train` command using the given input dataset
```

Comment:
```suggestion
- Enhance `find` command to find configurations of supported tasks / algorithms / models / backbones
- Introduce `build` command to customize task or model configurations in isolated workspace
- Auto-config feature to automatically select the right algorithm and default model for the `train` & `build` command by detecting the task type of given input dataset
```
Could you move last 2 items to new features?

Reply:
Revised.